std/format/read.d

```diff
@@ -1,35 +1,69 @@
 // Written in the D programming language.
 
 /**
-   This is a submodule of $(MREF std, format).
-   It provides some helpful tools.
-
-   Copyright: Copyright The D Language Foundation 2000-2013.
-
-   License: $(HTTP boost.org/LICENSE_1_0.txt, Boost License 1.0).
-
-   Authors: $(HTTP walterbright.com, Walter Bright), $(HTTP erdani.com,
-   Andrei Alexandrescu), and Kenji Hara
-
-   Source: $(PHOBOSSRC std/format/read.d)
+This is a submodule of $(MREF std, format).
+
+It provides two functions for reading formatted $(LREF unformatValue)
+reads a single value, while $(LREF formattedRead) reads several
+values at once and matches the characters found between format
+specifiers.
+
+For reading purpose, only part of the possibilities of $(MREF_ALTTEXT
+format strings, std,format) is used: The parameters are almost
+completely ignored. The only exception is a single $(B '*'): It
+denotes that the read value is not assigned to a variable, but is
+just discarded. A $(B '*') can only be applied to a $(B 'c'),
+$(B 'd') or $(B 'u') format character.
+
+A space outside of a format specifier has a special meaning: It
+matches any sequence of whitespace characters, not just a single
+space.
+
+The following combinations of format characters and types are
+available:
+
+$(BOOKTABLE ,
+$(TR $(TH) $(TH s) $(TH c) $(TH d, u, b, o, x, X) $(TH e, E, f, g, G) $(TH r) $(TH compound))
+$(TR $(TD `bool`) $(TD yes) $(TD $(MDASH)) $(TD yes) $(TD $(MDASH)) $(TD $(MDASH)) $(TD $(MDASH)))
+$(TR $(TD `null`) $(TD yes) $(TD $(MDASH)) $(TD $(MDASH)) $(TD $(MDASH)) $(TD $(MDASH)) $(TD $(MDASH)))
+$(TR $(TD $(I integer)) $(TD yes) $(TD $(MDASH)) $(TD yes) $(TD $(MDASH)) $(TD yes) $(TD $(MDASH)))
+$(TR $(TD $(I floating point)) $(TD yes) $(TD $(MDASH)) $(TD $(MDASH)) $(TD yes) $(TD yes) $(TD $(MDASH)))
+$(TR $(TD $(I character)) $(TD yes) $(TD yes) $(TD yes) $(TD $(MDASH)) $(TD $(MDASH)) $(TD $(MDASH)))
+$(TR $(TD $(I string)) $(TD yes) $(TD $(MDASH)) $(TD $(MDASH)) $(TD $(MDASH)) $(TD $(MDASH)) $(TD yes))
+$(TR $(TD $(I array)) $(TD yes) $(TD $(MDASH)) $(TD $(MDASH)) $(TD $(MDASH)) $(TD $(MDASH)) $(TD yes))
+$(TR $(TD $(I associative array)) $(TD yes) $(TD $(MDASH)) $(TD $(MDASH)) $(TD $(MDASH)) $(TD $(MDASH)) $(TD yes))
+)
+
+The examples below show how to use these combinations with $(LREF
+unformatValue). Everything explained there can also be applied to
+$(LREF formattedRead).
+
+Copyright: Copyright The D Language Foundation 2000-2013.
+
+License: $(HTTP boost.org/LICENSE_1_0.txt, Boost License 1.0).
+
+Authors: $(HTTP walterbright.com, Walter Bright), $(HTTP erdani.com,
+Andrei Alexandrescu), and Kenji Hara
+
+Source: $(PHOBOSSRC std/format/read.d)
  */
 module std.format.read;
 
-/// Booleans
+/// Booleans can be read as words `false` and `true` as well as as integers `0` and `1`.
```

Comment:
I think that the unittest is fairly straightforward, no extra explanations are needed. I suggest undoing this change.